### DISCUSSION TITLE
How can I remove the code about highlight.js(V1)

### DISCUSSION THREAD

**Discussion by quicklearning123** [facebook/docusaurus#4528] [Q&A]
There is a firewall here, so I can't visit this link (cloudflare.com)
I just want remove the code about highlight.js like this:
```
    <link
      rel="stylesheet"
      href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css"
    />
```

**Comment by slorber** [comment]
There is a "highlight.themeUrl" in v1 config: https://v1.docusaurus.io/docs/en/site-config#highlight
